src/screen.cpp

```diff
@@ -300,10 +309,12 @@ void Screen::setVisible(bool visible) {
     if (mVisible != visible) {
         mVisible = visible;
 
-        if (visible)
-            glfwShowWindow(mGLFWWindow);
-        else
-            glfwHideWindow(mGLFWWindow);
+        if (mOwningGLFWContext) {
```

Comment:
Why disable setVisible() for non-owned GLFW context? I think it would be quite reasonable to leave this on.